### PR TITLE
[Tests/Src] Add positive test cases for unit testing 

### PR DESCRIPTION
**Pull request by wooksong**
This patch adds some positive test cases for unit testing of tensor_ros_src and tensor_ros2_src.

Signed-off-by: Wook Song <wook16.song@samsung.com>

### CODE REVIEW

File: tests/tensor_ros_src/runTest.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+##
+## @file runTest.sh
+## @author Wook Song <wook16.song@gmail.com>
+## @date  2월 28 2019
```
Comment (leemgs) [inline tests/tensor_ros_src/runTest.sh]: s/2월 28 2019/Feb-28-2019

File: tests/tensor_ros_src/runTest.sh [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+## @brief SSAT test cases for NNStreamer-ros
+#
+if [[ "$SSATAPILOADED" != "1" ]]
+then
+	SILENT=0
+	INDEPENDENT=1
+	search="ssat-api.sh"
+	source $search
+	printf "${Blue}Independent Mode${NC}
+"
```
Comment (leemgs) [inline tests/tensor_ros_src/runTest.sh]: Do you want to add **"** at the new line?

File: tests/tensor_ros_src/runTest.sh (hunk 1 above)
Comment (wooksong) [reply tests/tensor_ros_src/runTest.sh]: Ah.....auto-generation fails again.

File: tests/tensor_ros_src/runTest.sh (hunk 2 above)
Comment (wooksong) [reply tests/tensor_ros_src/runTest.sh]: Nope. Thanks, I will fix it.

**Review by again4you — APPROVED** [review]
LGTM!

**Review by leemgs — APPROVED** [review]
LGTM

**Pull request merged**